java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -301,7 +302,6 @@ disabled.modules=\
     org.netbeans.modules.team.commons,\
     org.netbeans.modules.team.ide,\
     org.netbeans.modules.templates,\
-    org.netbeans.modules.terminal,\
```

Comment:
We do not need it because of the terminal. We need it because of dependency on `dlight.nativeexecution`. We use `org.netbeans.modules.nativeexecution.pty` in `CPPLiteDebugger`. And `org.netbeans.modules.nativeexecution.pty.IOConnector` uses `org.netbeans.modules.terminal.api.*`

A solution can be to extract PTY-related stuff that happens to be used by cpplite from `dlight.nativeexecution` into a separate module. It's ugly, however, that only a part of the PTY API can be extracte, because a part of it depends on the `IOConnector`, which uses `org.netbeans.modules.terminal.api`. We would need to extract that API also.

Reply:
Let's record the issue in JIRA and let's leave it for some future version. The `PTY` interface could for example be moved to `extexecution` API.

Reply:
I've created NETBEANS-5516